### PR TITLE
Fix #7: Add chart detail page

### LINKED ISSUE #7: Large chart view
When a chart is clicked, it should appear bigger (almost taking up the entire viewport) with other information and controls (description, options, etc.) shown on the side.

Mock from @spasovski:

A mock of a page showing a large single chart with options abovve the chart and a description below it

### PR DESCRIPTION
**Pull request by openjck**

### CODE REVIEW

**Comment by openjck** [comment]
Click on any chart to reach the chart detail page

**Comment by openjck** [comment]
This looks more complicated than it really is. About half the changes involve reworking the filter sidebar so that we can enable it on a page-by-page basis. The detail page itself is pretty simple.

File: distributionviewer/core/static/js/app/components/views/filtered-charts.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+export class FilteredCharts extends React.Component {
+  constructor(props) {
+    super(props);
+    this.filteredClass = ' filtered';
+  }
+
+  componentDidMount() {
+    var wrapper = document.getElementsByClassName('global-wrapper')[0];
+    wrapper.className += this.filteredClass;
```
Comment (spasovski) [inline distributionviewer/core/static/js/app/components/views/filtered-charts.js]: `wrapper.classList.add('filtered')` would let us forego the space prefixed string.

File: distributionviewer/core/static/js/app/components/views/filtered-charts.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  }
+
+  componentWillUnmount() {
+    var wrapper = document.getElementsByClassName('global-wrapper')[0];
+    wrapper.className = wrapper.className.replace(this.filteredClass, '');
```
Comment (spasovski) [inline distributionviewer/core/static/js/app/components/views/filtered-charts.js]: `wrapper.classList.remove(this.filteredClass)`

**Comment by spasovski** [comment]
r+wc thanks

I'm working on showing these with the sample architecture data we have in db.json instead.

File: distributionviewer/core/static/js/app/components/views/filtered-charts.js (hunk 1 above)
Comment (openjck) [inline distributionviewer/core/static/js/app/components/views/filtered-charts.js]: Good catch. Sounds good to me as long as we don't need to support IE < 11.

http://caniuse.com/#search=classlist

**Comment by openjck** [comment]
> I'm working on showing these with the sample architecture data we have in db.json instead.

Awesome, that's exciting!

Updated with `classList`, self-merging...

**Pull request merged**